src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -178,6 +190,12 @@ export class NetCoreTaskHelper implements TaskHelper {
         return false;
     }
 
+    public static async isWebApp(appProject: string): Promise<boolean> {
+        const projectContents = await fse.readFile(appProject);
+
+        return /Microsoft\.NET\.Sdk\.Web/i.test(projectContents.toString());

```

Comment:
`/<\s*Project\s+Sdk=\"Microsoft.NET.Sdk.Web\"/` ?

(a bit more specific)

Reply:
I thought about it, but we should maybe allow for some other things, like:
`<Project DefaultTargets="build" ToolsVersion="1234" Sdk="...">`, since there can be more things than just `Sdk` as attributes.

Maybe `/<\s*Project\s+.*Sdk\s*=\s*\"Microsoft.NET.Sdk.Web\"/`? Although maybe it's too complicated...

Reply:
Good point. Also the `<Project>` element can span multiple lines. Maybe checking just for `Sdk\s*=\s*\"Microsoft.NET.Sdk.Web\"` is a good compromise...